src/status_im/protocol/web3/keys.cljs

```diff
@@ -19,3 +19,6 @@
       (fn [err res]
         (swap! password->keys assoc password res)
         (callback res)))))
+
+(defn reset-keys! []
+  (reset! password->keys {}))
```

Comment:
Not part of PR I guess, but how come password->keys is a separate atom from app-db? To avoid re-frisk leak or something? Seems a bit weird to me

Reply:
1. The whole `protocol` lives it's own mutable life atm. 
2. Also this part of state is not and will not be useful in any part of ui or event handling.
3. If you will take a look at `generateSymKeyFromPassword` usage now, it uses hardcoded passwords https://github.com/status-im/status-react/blob/6b66d3c4994748c0510b842d777ba6e7a65ed06d/src/status_im/protocol/web3/keys.cljs#L3, so this part of code was mostly temporal dirty workaround
4. which should be reworked from the very beginning when we finally get to double ratchet
5. which is another reason for keeping `protocol` with it's own state and stuff decoupled from other parts of app and app-data in particular